app/code/Magento/Checkout/etc/frontend/di.xml

```diff
@@ -59,6 +59,7 @@
                 <item name="totalsSortOrder" xsi:type="object">Magento\Checkout\Block\Checkout\TotalsProcessor</item>
                 <item name="directoryData" xsi:type="object">Magento\Checkout\Block\Checkout\DirectoryDataProcessor</item>
             </argument>
+            <argument name="serializer" xsi:type="object">\Magento\Framework\Serialize\Serializer\JsonHexTag</argument>
```

Comment:
Please remove the first slash from the class name to be consistent with other declarations